xml/Microsoft.Extensions.Caching.Distributed/DistributedCacheExtensions.xml

```diff
@@ -89,7 +89,7 @@
         <param name="key">The key to get the stored data for.</param>
         <summary>Asynchronously gets a string from the specified cache with the specified key.</summary>
         <returns>A task that gets the string value from the stored cache key.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>This method stores all the exceptions that its synchronous counterpart can throw in the task it returns. If an exception is stored during execution, it will be thrown when you await either this method or the task it returns. For the stored exceptions, see the exceptions thrown by <see cref="M:Microsoft.Extensions.Caching.Distributed.DistributedCacheExtensions.GetString(Microsoft.Extensions.Caching.Distributed.IDistributedCache,System.String)" />.</remarks>
```

Comment:
> This method stores all the exceptions that its synchronous counterpart can throw in the task it returns.

I think we should caveat the "all" slightly in the boilerplate... typically we allow/encourage usage exceptions (in particular ArgumentException and its derived types) to escape synchronously from asynchronous methods.  So we might want to make this say something more like "This method stores all non-usage exceptions...".

> If an exception is stored during execution, it will be thrown when you await either this method or the task it returns.

"awaiting a method" isn't actually a thing: you're always using await with some instance, and if you write "await Foo()", that instance is whatever Foo() returns.
https://devblogs.microsoft.com/pfxteam/invoke-the-method-with-await-ugh/
so this should just say something like "it will be thrown when you await the task the method returns".